### PR TITLE
Portland 2022 full website and ticket sales blog post

### PR DESCRIPTION
**Pull request by thatdocslady**
- [x] Verify ticket prices 
- [x] Decide if we want to extend the CFP 
- [ ] Fix schedule page @ericholscher  
- [x] Publish Tito site 
- [x] Open grant form @mxsasha 
- [x] Update CFP in Pretalx

### CODE REVIEW

**Comment by mxsasha** [comment]
Before we merge this we also need to set the tito to live (links should work if you're logged in), and switch the grant form to open.

**Comment by thatdocslady** [comment]
> Before we merge this we also need to set the tito to live (links should work if you're logged in), and switch the grant form to open.

I added those tasks to the checklist, thanks!

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (thatdocslady) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]:
```suggestion
.. post:: Feb 2, 2022
```

File: docs/_data/portland-2022-config.yaml
Comment (thatdocslady) [inline docs/_data/portland-2022-config.yaml:234]: @plaindocs is this for just the sign-up sheet? I may have incorrectly flipped this...

File: docs/_data/portland-2022-config.yaml
Comment (thatdocslady) [inline docs/_data/portland-2022-config.yaml:225]: The hike flag is on under "truthy things that don't change" and there are mentions of it on the website, but I don't know whether to delete the actual text or turn off the flag. Halp!

File: docs/conf/portland/2022/team.rst
Comment (thatdocslady) [inline docs/conf/portland/2022/team.rst]: I tried extracting this information to an included page but the preview build isn't pulling it for some reason :(

**Review by thatdocslady — COMMENTED** [review]
General question: I noticed the buttons for tickets and CFP are too close together.. if someone more savvy could look at it I'd appreciate it!

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (thatdocslady) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]:
```suggestion
In addition to the online resources, the Write the Docs East Coast Quorum meetup will hold a `Conference Proposal Workshop on February 10th in US Eastern time zone <https://www.meetup.com/virtual-write-the-docs-east-coast-quorum/events/283502054/>`_. Join your fellow documentarians to discuss potential proposals, brainstorm ideas, and get feedback on your proposal before you submit. Big shout-out to Alyssa Rock and the organizers of this Quorum meetup!
```

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (thatdocslady) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]:
```suggestion
.. _virtual sponsorship prospectus: https://www.writethedocs.org/conf/portland/2022/sponsors/prospectus/
```

**Comment by mxsasha** [comment]
> General question: I noticed the buttons for tickets and CFP are too close together.. if someone more savvy could look at it I'd appreciate it!

I think the issue here is that our CSS only expects two buttons, not three.

**Comment by thatdocslady** [comment]
> > General question: I noticed the buttons for tickets and CFP are too close together.. if someone more savvy could look at it I'd appreciate it!
> 
> I think the issue here is that our CSS only expects two buttons, not three.

See but at the bottom of the page we have the same buttons and the look fine =/ @ericholscher any CSS magic that we can do here?

**Comment by ericholscher** [comment]
@thatdocslady I'd just make the text fit so the buttons look OK. We generally only have 2 callouts, and I think that's a reasonable number. I think we normally hide the tickets until after the CFP closes.

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (ericholscher) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]: This feels odd given we've had many virtual conferences..

```suggestion
If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_! We are still planning to limit capacity of the event to ensure that we have a good experience for everyone, but normally that happens 1-2 weeks before the conference date.
```

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (ericholscher) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]: A bit more clear about not having individual Q&A

```suggestion
* Excellent single track of pre-recorded presentations with moderated Q&A of speakers
```

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (ericholscher) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]:
```suggestion
* A social event, but you'll need to bring your own beverage and snacks :)
```

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (ericholscher) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]:
```suggestion
We plan the event to ensure lots of interaction, which is the main reason we're all gathered together, and we think it will be another great year of helping our community connect.
```

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (ericholscher) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]:
```suggestion
Submit your talk by **{{ cfp.ends }}**
```

File: docs/conf/portland/2022/news/tickets-on-sale.rst
Comment (ericholscher) [inline docs/conf/portland/2022/news/tickets-on-sale.rst]:
```suggestion
We are continuing our grant program again this year in a similar format. 
```

File: docs/conf/portland/2022/opportunity-grants.rst
Comment (ericholscher) [inline docs/conf/portland/2022/opportunity-grants.rst:22]: 🤯

File: docs/conf/portland/2022/team.rst
Comment (ericholscher) [reply docs/conf/portland/2022/team.rst]: Seems it changes each year? I don't think this works well for an include. 

It looks like the include file is empty though, which is likely the issue.

**Review by ericholscher — COMMENTED** [review]
Great start!

File: docs/_data/portland-2022-config.yaml
Comment (plaindocs) [reply docs/_data/portland-2022-config.yaml:225]: I think False works fine.

**Comment by thatdocslady** [comment]
@ericholscher I see the changes to the schedule page and they look good except the writing day isn't visible and it's not in the source file either.... should I hardcode something in the meantime? I don't know if it's worth adding an IF statement for writing day and all the flag shenanigans :D

File: docs/_data/portland-2022-config.yaml
Comment (plaindocs) [inline docs/_data/portland-2022-config.yaml]: Aren't we doing these?

File: docs/include/conf/portland/team.rst
Comment (plaindocs) [inline docs/include/conf/portland/team.rst]: I don't think it makes sense to include this. 

The bios are already included, and this way whenever we change it next year it'll update other years also.

Comment (thatdocslady) [reply docs/include/conf/portland/team.rst]: I guess I was trying to create a situation where each conference has a different include and it magically changes for Portland, Prague, etc. But now I see that I'm probably missing an IF statement somewhere :) if it's too much of a PITA I can just revert it back to hardcoded content...

File: docs/_data/portland-2022-config.yaml
Comment (thatdocslady) [reply docs/_data/portland-2022-config.yaml]: I confess that I don't really know what this flag does. It was false and I flipped it to true and then I checked the website and noticed some links to the sign-up sheet (which doesn't work now) so I flipped it back to false thinking it'll solve it.... I will happily defer to whatever is needed here.

File: docs/include/conf/portland/team.rst
Comment (plaindocs) [reply docs/include/conf/portland/team.rst]: yeah, even that way though, it'll break. I reckon leave this one hard coded.

As an alternative, :smiling_imp: `conf.yaml`

```yaml
roles:
  chair: eric
  speakers: sam
```

but I don't _think_ it's worth it for this.

File: docs/_data/portland-2022-config.yaml
Comment (plaindocs) [reply docs/_data/portland-2022-config.yaml]: Eh, maybe I'm confused on this one. It is indeed listed in things that *do* change.

File: docs/conf/portland/2022/team.rst
Comment (ericholscher) [inline docs/conf/portland/2022/team.rst:76]: 💯 Thanks for updating this.

**Review by ericholscher — APPROVED** [review]

**Pull request merged**
